.eslintrc

```diff
@@ -1,131 +1,6 @@
-        "no-implied-eval": 2,
-        "no-inner-declarations": [0, "functions"],
-        "no-invalid-regexp": 2,
-        "no-iterator": 2,
-        "no-label-var": 2,
-        "no-labels": 2,
-        "no-lone-blocks": 2,
-        "no-lonely-if": 2,
-        "no-loop-func": 2,
-        "no-mixed-requires": [2, false],
-        "no-multi-str": 2,
-        "no-native-reassign": 2,
-        "no-negated-in-lhs": 2,
-        "no-nested-ternary": 0,
-        "no-new": 2,
-        "no-new-func": 2,
-        "no-new-object": 2,
-        "no-new-require": 2,
-        "no-new-wrappers": 2,
-        "no-obj-calls": 2,
-        "no-octal": 2,
-        "no-octal-escape": 2,
-        "no-path-concat": 0,
-        "no-plusplus": 0,
-        "no-process-exit": 0,
-        "no-proto": 2,
-        "no-redeclare": 2,
-        "no-regex-spaces": 2,
-        "no-restricted-modules": 0,
-        "no-return-assign": 2,
-        "no-script-url": 2,
-        "no-self-compare": 0,
-        "no-sequences": 2,
-        "no-shadow": 0,
-        "no-shadow-restricted-names": 2,
-        "no-spaced-func": 2,
-        "no-space-before-semi": 2,
-        "no-sparse-arrays": 2,
-        "no-sync": 0,
-        "no-ternary": 0,
-        "no-trailing-spaces": 2,
-        "no-undef": 2,
-        "no-undefined": 0,
-        "no-undef-init": 2,
-        "no-underscore-dangle": 0,
-        "no-unreachable": 2,
-        "no-unused-expressions": 2,
-        "no-unused-vars": [2, {"vars": "all", "args": "after-used"}],
-        "no-use-before-define": 0,
-        "no-warning-comments": [0, { "terms": ["todo", "fixme", "xxx"], "location": "start" }],
-        "no-with": 2,
-        "no-wrap-func": 2,
-        "no-mixed-spaces-and-tabs": [2, false],
-        "block-scoped-var": 0,
-        "brace-style": [2, "1tbs"],
-        "camelcase": 0,
-        "complexity": [0, 11],
-        "consistent-return": 0,
-        "consistent-this": [2, "self"],
-        "curly": [0, "all"],
-        "default-case": 0,
-        "dot-notation": 2,
-        "eol-last": 2,
-        "eqeqeq": 2,
-        "func-names": 0,
-        "func-style": [2, "declaration"],
-        "guard-for-in": 0,
-        "max-depth": [0, 4],
-        "max-len": [2, 80, 4],
-        "max-nested-callbacks": [2, 5],
-        "max-params": [0, 3],
-        "max-statements": [0, 10],
-        "handle-callback-err": 2,
-        "new-cap": 0,
-        "new-parens": 0,
-        "one-var": 0,
-        "quote-props": 0,
-        "quotes": [2, "single"],
-        "radix": 0,
-        "semi": 2,
-        "sort-vars": 2,
-        "space-after-keywords": [2, "always"],
-        "space-in-brackets": [2, "never"],
-        "space-infix-ops": 2,
-        "space-return-throw-case": 2,
-        "strict": 0,
-        "use-isnan": 2,
-        "valid-jsdoc": 0,
-        "valid-typeof": 2,
-        "wrap-iife": 2,
-        "wrap-regex": 0,
-        "yoda": [2, "never"]
-    }
+  "extends": "strongloop",
+  "rules": {
+    "comma-dangle": 0
```

Comment:
`always-multiline` is the default being opted out of. I was trying to avoid all the diff noise, otherwise I'd be more than happy to fix all 47 errors it produces.